src/react/utils/__tests__/index.test.jsx

```diff
@@ -0,0 +1,143 @@
+            error: {
+                status: 409,
+            },
+            expectedErrorMessage: errorMessages[4],
+        },
+        {
+            it: `should return the error message: "${errorMessages[5]}" if status code is 500`,
+            error: {
+                status: 500,
+            },
+            expectedErrorMessage: errorMessages[5],
+        },
+        {
+            it: `should return the error message: "${errorMessages[5]}" if status code is 503`,
+            error: {
+                status: 503,
+            },
+            expectedErrorMessage: errorMessages[5],
+        },
+        {
+            it: 'should return the error message contains in error.message if exists',
+            error: {
+                message: errorMessages[6],
+            },
+            expectedErrorMessage: errorMessages[6],
+        },
+        {
+            it: 'should return the status code if no message exists in the error object and status code is not handled',
+            error: {
+                status: 501,
+            },
+            expectedErrorMessage: 'Failed with error status: 501',
+        },
+    ];
+
+    tests.forEach(t => {
+        it(`${t.it}`, () => {
+            expect(errorParser(t.error).message).toBe(t.expectedErrorMessage);
+        });
+    });
+
+    it('should return a formatted date', () => {
+        const date = 'Mon Oct 12 2020 17:28:03';
+
+        expect(formatDate(new Date(date))).toBe(date);
+    });
+
+    it('should return a string formatted which reflect the size', () => {
+        expect(formatBytes(0)).toBe('0 Bytes');
+        expect(formatBytes(1)).toBe('1 Bytes');
+        expect(formatBytes(1024)).toBe('1 KB');
+        expect(formatBytes(10240000)).toBe('9.77 MB');
+        expect(formatBytes(10240000000)).toBe('9.54 GB');
+        expect(formatBytes(10240000000000)).toBe('9.31 TB');
+        expect(formatBytes(10240000000000000)).toBe('9.09 PB');
+        expect(formatBytes(10240000000000000000)).toBe('8.88 EB');
+        expect(formatBytes(10240000000000000000000)).toBe('8.67 ZB');
+        expect(formatBytes(10240000000000000000000000)).toBe('8.47 YB');
+        expect(formatBytes(10240000000000000000000000, 1)).toBe('8.5 YB');
+    });
+
+    const urls = ['/buckets/', '/buckets', '/buckets/bucket1', '/buckets/bucket1/test/'];
+
+    it('should return a string with a trailing slash removed at the end if exists', () => {
+        urls.forEach(url => {
+            expect(stripTrailingSlash(url).slice(-1)).not.toBe('/');
+        });
+        expect(stripTrailingSlash('').slice(-1)).not.toBe('/');
+    });
+
+    it('should return a string with a trailing slash added at the end if no exists', () => {
+        urls.forEach(url => {
+            expect(addTrailingSlash(url).slice(-1)).toBe('/');
+        });
+        expect(addTrailingSlash('').slice(-1)).toBe('');
+    });
+
+    it('should return a string which pluralize the noun taken in params in count > 1', () => {
```

Comment:
Can we test the third parameter as well?

Reply:
done